support/overlay/overlay.nix

```diff
@@ -63,6 +63,15 @@ in
 
     gxlimg = callPackage ./gxlimg { };
 
+    meson64-tools = let
+      src = final.fetchFromGitHub {
+        owner = "angerman";
+        repo = "meson64-tools";
+        rev = "a2d57d11fd8b4242b903c10dca9d25f7f99d8ff0";
+        sha256 = "Wyx4ngfDN6jSEBwrgH8z44wntJEKMuCQz56MrU9mB5E=";
+      };
+    in (import src { nixpkgs = final; }).meson64-tools;
+
```

Comment:
We shouldn't do IFDs; they'll cause issues when we get some CD done through Hydra.